nephio-ansible-install/roles/nephio/install/tasks/packages_files.yaml

```diff
@@ -0,0 +1,17 @@
+## © 2022 Nokia
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
+---
+- name: Download the {{ item.package }} packages, with url {{ nephio.packages_url }}/{{ item.package }}
+  shell: kpt pkg get --for-deployment {{ nephio.packages_url }}/{{ item.package }}
+  args: 
+    chdir: "{{ tmp_directory }}/{{ nephio.install_dir}}"
+  register: result
+  failed_when:
+    - result.rc > 1
+    - result.rc == 1 and "already exists" not in result.stderr
+
+- name: Initialize {{ item.package }}
```

Comment:
FYI I ran the playbook locally on the VM via the command

```
ansible-playbook playbooks/install-prereq.yaml --connection=local
```

and ran into the below issue for this taks

```
TASK [nephio/install : Initialize nephio-configsync] ********************************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": "kpt fn render nephio-configsync", "delta": "0:00:00.212946", "end": "2022-12-13 01:41:46.003805", "msg": "non-zero return code", "rc": 1, "start": "2022-12-13 01:41:45.790859", "stderr": "Package \"nephio-configsync\": \n[RUNNING] \"gcr.io/kpt-fn/apply-replacements:v0.1.1\"\n[FAIL] \"gcr.io/kpt-fn/apply-replacements:v0.1.1\" in 0s\nError: exit status 1\ndocker must be running to use this command\nTo install docker, follow the instructions at https://docs.docker.com/get-docker/.\n ", "stderr_lines": ["Package \"nephio-configsync\": ", "[RUNNING] \"gcr.io/kpt-fn/apply-replacements:v0.1.1\"", "[FAIL] \"gcr.io/kpt-fn/apply-replacements:v0.1.1\" in 0s", "Error: exit status 1", "docker must be running to use this command", "To install docker, follow the instructions at https://docs.docker.com/get-docker/.", " "], "stdout": "", "stdout_lines": []}
```

I had to log out of the VM and re-login to run the playbook again to go through all the roles/tasks in install_prereq.yaml playbook.

Reply:
we need to add some text in the readme to use connection-local. This indeed requires you to login and logout again.